spec/controllers/admin/projects_controller_spec.rb

```diff
@@ -121,5 +121,16 @@
       end
     end
   end
-end
 
+  describe "POST move_project_to_channel" do
+    let (:project) { FactoryGirl.create(:project) }
+    let (:channel) { FactoryGirl.create(:channel) }
+
+    it "shouldn't have a channel" do
```

Comment:
@matheuscla nem parece que te ajudei cara =P.
Essa descrição não tem nada haver com o que você ta testando, tinah que ser o contrário. Algo assim `adds project to channel `.

E ta faltando o teste do seu service. Amanhã de manhã bora fazer um intensivão de testes.